### DISCUSSION TITLE
Detect and execute upon Dropbox links in Fanbox posts

### DISCUSSION THREAD

**Discussion by teward** [KJHJason/Cultured-Downloader#62] [Ideas]
The current code is configured to detect MEGA links and "other download links".  However, it doesn't seem to be able to parse and detect "masked" Google Drive links, which are embedded with `[ Download Now ]` or similar text.

It'd be great if Fanbox "Detect other links (like MEGA)" would actually be capable of detecting and acting upon Dropbox links.

**Comment by KJHJason** [comment]
Oops, I forgot to reply to this but this _should be resolved_ in the upcoming version 5 of Cultured Downloader as it will be using the API of the various platforms instead of web scraping.

**Comment by KJHJason** [comment]
Hello @teward, I have just released V5.0.0. Let me know if you are still facing this issue.

**Reply by teward** [reply]
Will take some time, as I no longer have been using this tool to pull from fanbox and other locations.  I'll let you know when I do get around to testing it.  Unless someone else wants to test it :)

**Reply by KJHJason** [reply]
Sure, no worries!

**Comment by teward** [comment]
@KJHJason hard failure problem that prevents even *testing* this.  Issue #271 where 5.1.0 says it can't find the cookie from the TXT/JSON dumps or that the value when extracted and put into its option in settings for pixiv fanbox is invalid. Pixiv main was accepted, but the fanbox session cookies were not.
